pyuvdata/uvdata/uvdata.py

```diff
@@ -2415,7 +2587,10 @@ def unphase_to_drift(self, phase_frame=None, use_ant_pos=False):
                 / const.c.to("m/s").value
                 * self.freq_array.reshape(1, self.Nfreqs)
             )
-            phs = np.exp(-1j * 2 * np.pi * (-1) * w_lambda[:, None, :, None])
+            if self.future_array_shapes:
+                phs = np.exp(-1j * 2 * np.pi * (-1) * w_lambda[:, :, None])
+            else:
+                phs = np.exp(-1j * 2 * np.pi * (-1) * w_lambda[:, None, :, None])
```

Comment:
Not an actual problem, but there's a good chunk of the `phase` method (and a few others related to w-projection tasks) that's been updated in PR #979 -- would be good to make sure that our two PRs don't inadvertently collide! 😅

Reply:
Yeah, I suspect we'll have to merge one and then adjust the other after a rebase. If we have good testing coverage I think we'll catch things but we'll need to be careful.